api/app/Http/Middleware/TransformInputDataMiddleware.php

```diff
@@ -27,8 +27,11 @@ public function handle($request, Closure $next)
             // Find any potential camelCase keys in the 'root' array, and convert
             // them to snake_case
             if (!ctype_lower($key)) {
-                $request->offsetSet(snake_case($key), $value);
-                $request->offsetUnset($key);
+                // Only convert if the key will change
+                if ($key != snake_case($key)) {
```

Comment:
Actually I just tested it:
```
[feature/social-login][zak@zeroth:~/sites/spira/spira]
[15:51:13] $ php -r "var_dump(ctype_lower('oauth_token'));"
bool(false)

[feature/social-login][zak@zeroth:~/sites/spira/spira]
[15:51:28] $ php -r "var_dump(ctype_lower('oauthtoken'));"
bool(true)
```
So it seems that it considers the underscore to be not a lowercase letter. TIL